packages/scratch-gui/src/lib/project-saver-hoc.jsx

```diff
@@ -427,7 +427,8 @@ const ProjectSaverHOC = function (WrappedComponent) {
             isManualUpdating: getIsManualUpdating(loadingState),
             loadingState: loadingState,
             locale: state.locales.locale,
-            onUpdateProjectThumbnail: ownProps.onUpdateProjectThumbnail ?? storage.saveProjectThumbnail,
+            onUpdateProjectThumbnail: ownProps.onUpdateProjectThumbnail ??
+                ((projectId, thumbnail) => storage.saveProjectThumbnail(projectId, thumbnail)),
```

Comment:
Previously this was `undefined` if the storage didn't have the `storage.saveProjectThumbnail` function. Now it's a function that will throw an error because `storage.saveProjectThumbnail` doesn't exist. Please handle that case.